.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,43 @@
 <!-- Please review the following before submitting a PR:
 Che's Contributing Guide: https://github.com/eclipse/che/blob/master/CONTRIBUTING.md
 Pull Request Policy: https://github.com/eclipse/che/wiki/Development-Workflow#pull-requests
-
-COMMITTERS: please include labels on each PR. Labels are listed here: https://github.com/eclipse/che/wiki/Labels but at a minimum you should include `kind/..` and `Dev Open Pull Request Status` labels.
 -->
 
 ### What does this PR do?
 
 
+### Screenshot/screencast of this PR
+<!-- Please include a screenshot or a screencast explaining what is doing this PR -->
+
+
 ### What issues does this PR fix or reference?
+<!-- Please include any related issue from eclipse che repository (or from another issue tracker).
+     Include link to other pull requests like documentation PR from [the docs repo](https://github.com/eclipse/che-docs)
+-->
+
+
+### How to test this PR?
+<!-- Please explain for example :
+  - The test platform (openshift, kubernetes, minikube, CodeReady Container, docker-desktop, etc)
+  - Installation method: chectl / che-operator
+  - steps to reproduce
+ -->
+
 
-<!-- #### Changelog -->
-<!-- The changelog will be pulled from the PR's title. 
-     Please provide a clear and meaningful title to the PR and don't include issue number -->
+### PR Checklist
 
+As the author of this Pull Request I made sure that:
 
-#### Release Notes
-<!-- markdown to be included in marketing announcement - N/A for bugs -->
+- [ ] The Eclipse Contributor Agreement is valid
+- [ ] Code produced is complete
```

Comment:
what does it mean that "code is complete"? Is it defined somewhere?

Reply:
not in work in progress anymore

Reply:
It's good to have an explanation section here, something like we have for 'How to test this PR?'.

Reply:
ok, so it's something the author decides when the PR is complete. We already have WIP bot and Draft PR for unfinished state. But I think it's still good to have it here.

Maybe rename it or add little explanation, sth like `Code complete (don't plan to do any changes except review fixes)`.

Reply:
I think that applies to all the check points: we should provide a detailed and clear description and provide a link to it. I will draft something in the wiki.

Reply:
description has been added